src/main/java/io/akka/ai/application/OpenAiClient.java

```diff
@@ -0,0 +1,65 @@
+package io.akka.ai.application;
+
+import akka.javasdk.JsonSupport;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.openai.client.OpenAIClient;
+import com.openai.client.okhttp.OpenAIOkHttpClient;
+import com.openai.models.ChatModel;
+import com.openai.models.responses.ResponseCreateParams;
+import io.akka.tagging.domain.HospitalizationTag;
+import io.akka.tagging.domain.TaggingResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.CompletionStage;
+
+public class OpenAiClient implements AIClient {
+
+  private static final Logger log = LoggerFactory.getLogger(OpenAiClient.class);
+  private final OpenAIClient client;
+
+  record OpenAiTaggingResult(
+    HospitalizationTag tag,
+    int confidencePercentage,
+    String reasoning
+  ) {
+  }
+
+  public OpenAiClient() {
+    var key = System.getenv("OPENAI_API_KEY");
```

Comment:
maybe verify that this is defined already in the Setup constructor, as we do in ask-akka-agent